src/ripple/core/SociDB.h

```diff
@@ -143,8 +150,8 @@ class Checkpointer
     The Checkpointer contains references to the session and job queue
     and so must outlive them both.
  */
-std::unique_ptr<Checkpointer>
-makeCheckpointer(soci::session&, JobQueue&, Logs&);
+std::shared_ptr<Checkpointer>
+makeCheckpointer(std::uintptr_t id, soci::session&, JobQueue&, Logs&);
```

Comment:
Nit: capitalize checkpointer in comment.

Reply:
Fixed in `50335297d [fold] Longer variable names`